docker-compose.yml

```diff
@@ -364,18 +392,51 @@ services:
       - _APP_DB_USER
       - _APP_DB_PASS
       - _APP_FUNCTIONS_TIMEOUT
+      - _APP_EXECUTOR_SECRET
+      - _APP_USAGE_STATS
+      - DOCKERHUB_PULL_USERNAME
+      - DOCKERHUB_PULL_PASSWORD
+
+  appwrite-executor:
+    container_name: appwrite-executor
+    entrypoint:
+      - php
+      - -e
+      - /usr/src/code/app/executor.php
+      - -dopcache.preload=opcache.preload=/usr/src/code/app/preload.php
+    stop_signal: SIGINT
+    build:
+      context: .
+      args: 
+        - DEBUG=false
+        - TESTING=true
+        - VERSION=dev
+    networks:
+      appwrite:
+      runtimes:
+    ports:
+      - 9519:80
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - appwrite-functions:/storage/functions:rw
+      - appwrite-builds:/storage/builds:rw
+      - /tmp:/tmp:rw
+    environment:
+      - _APP_ENV
+      - _APP_FUNCTIONS_TIMEOUT
+      - _APP_FUNCTIONS_BUILD_TIMEOUT
       - _APP_FUNCTIONS_CONTAINERS
       - _APP_FUNCTIONS_RUNTIMES
       - _APP_FUNCTIONS_CPUS
       - _APP_FUNCTIONS_MEMORY
       - _APP_FUNCTIONS_MEMORY_SWAP
-      - _APP_USAGE_STATS
-      - _APP_STATSD_HOST
-      - _APP_STATSD_PORT
-      - DOCKERHUB_PULL_USERNAME
-      - DOCKERHUB_PULL_PASSWORD
+      - _APP_EXECUTOR_SECRET
+      - _APP_EXECUTOR_RUNTIME_NETWORK
```

Comment:
Wan't most of this settings need to use `OPENRUNTIMES` as a prefix at some point?